### PR TITLE
set up CircleCI

### PR DESCRIPTION
**Pull request by afeld**
* Set some missing variables that I failed to copy over originally
* Add a check for the validity of the nginx config
* Remove TravisCI configuration (added by default by `ansible-galaxy init`)
* Run the role via the test playbook in CircleCI

## Follow-up TODOs

* [x] Re-trigger build for https://github.com/GSA/jenkins-deploy/pull/36, and remove `WORK IN PROGRESS` there if it passes
* [x] Remove `WORK IN PROGRESS` from this repository's description

### CODE REVIEW

**Comment by afeld** [comment]
The good news is, this failure is legitimate 🎉

File: tests/hosts [hunk 1]
```diff
@@ -0,0 +1 @@
+remote ansible_connection=docker
```
Comment (afeld) [inline tests/hosts:1]: This setting was a _huge_ discovery.

Comment (JJediny) [reply tests/hosts:1]: huh this is new to me to...

**Review by JJediny — APPROVED** [review]
LGTM

**Pull request merged**